run-dljc.sh

```diff
@@ -6,14 +6,16 @@
 ### Usage
 
 # - Move this script to an experiment directory.
-# - Make a file containing a list of git repositories, one per line. Repositories must be of the form: https://github.com/username/repository - the script is reliant on the number of slashes, so excluding https:// is an error.
+# - Make a file containing a list of git repositories, one per line.
```

Comment:
I can't comment on the `-i` documentation because it's not in the diff, but I found the "owned by you" text confusing.  Do you mean it's a fork?  I think the point is to run with small modifications.  You might want to connect the documentation of `-i` and `-u` to clarify both of them.

Also, "each line must be followed by" indicates that it's on a separate line.  I think you mean "at the end of that line".